### ISSUE TITLE
SA1649 does not work with file-scoped namespaces

### ISSUE THREAD

**Issue by mrzygood** [DotNetAnalyzers/StyleCopAnalyzers#3691]
Let's assume I have file `Foo.cs`.
In this file when `Bar class created like that it works fine and shows the rule violation:
```
namespace Example.App
{
    public class Bar
    {
    }
}
```

It does not show rule violation in this case:
```
namespace Example.App;

public class Bar
{
}
```

**Comment by sharwell** [comment]
Duplicate of #3435
